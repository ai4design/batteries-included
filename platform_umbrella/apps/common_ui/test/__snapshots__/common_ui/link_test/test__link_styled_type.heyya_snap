<a href="/" data-phx-link="redirect" data-phx-link-state="push" class="font-medium text-pink-500 hover:text-pink-600 hover:underline">
  Test Link
</a>