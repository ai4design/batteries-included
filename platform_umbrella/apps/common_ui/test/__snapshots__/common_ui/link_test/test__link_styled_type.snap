"<a href=\"/\" data-phx-link=\"redirect\" data-phx-link-state=\"push\" class=\"font-medium text-secondary-500 hover:text-secondary-600 hover:underline\">\n  Test Link\n</a>"